### ISSUE TITLE
Ability to hide keymapper?

### ISSUE THREAD

**Issue by jywarren** [publiclab/Leaflet.DistortableImage#258]
The latest release of LDI is now live at MapKnitter.org, congrats!!! I noticed that the keymapper help window stays open no matter what, and wondered if we could make it "dismissable"? It's super helpful, but after a while, seems like it's no longer needed. Thanks!

**Comment by rexagod** [comment]
@jywarren I did comment out the method that enables this since that wasn't included in the API at the time I posted the phase II PR, but no worries, I'll integrate this into the API!

https://github.com/publiclab/Leaflet.DistortableImage/blob/29590a2f3ac348beba1c5ac03f986af2f4778f1d/examples/select.html#L114-L115

**Comment by jywarren** [comment]
great, thank you! I think this is a bit of a priority as it can overlap
other things on the page right now. Much appreciated!!!

**Comment by rexagod** [comment]
@jywarren I've made the changes, just want you to have a look before I get to rebasing it. Thanks!

**Comment by jywarren** [comment]
Hi @rexagod @sashadev-sky is this resolved in the latest version? I believe we've turned off the keymapper in MapKnitter.org now via https://github.com/publiclab/mapknitter/pull/651, so we may just be waiting to ensure keymapper appears only once, and this issue is complete?

**Comment by sashadev-sky** [comment]
closed via #243 !
